Tests/UnitTests/Misc/ISODurationFormatterTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ISODurationTests.swift
+//
+//  Created by Facundo Menzella on 10/2/25.
+
+import Nimble
+import XCTest
+
+@testable import RevenueCat
+
+final class ISODurationFormatterTests: TestCase {
+
+    func testParseFullDuration() {
+        let durationString = "P1Y2M3W4DT5H6M7S"
+        guard let duration = ISODurationFormatter.parse(from: durationString) else {
```

Comment:
You can use [`XCTUnwrap`](https://developer.apple.com/documentation/xctest/xctunwrap(_:_:file:line:)) here. You'll need to make the test a throwing method, but it's fine for tests. And it makes the test cleaner.